src/bitmessagecurses/__init__.py

```diff
@@ -778,7 +782,8 @@ def sendMessage(sender="", recv="", broadcast=None, subject="", body="", reply=F
                         continue
                     if len(shared.connectedHostsList) == 0:
                         set_background_title(d, "Not connected warning")
-                        scrollbox(d, unicode("Because you are not currently connected to the network, "))
+                        scrollbox(d, unicode("Because you are not currently connected \
```

Comment:
(grammar-nazi-mode on) You shouldn't begin a sentence with "Because", use "Due to", "As", or change the order so that because is in middle.

Reply:
tell the original programmer ...   ;-)